src/components/v5/common/Pills/PermissionsBadge/types.ts

```diff
@@ -0,0 +1,5 @@
+import { PillsProps } from '../types';
+
+export interface PermissionsBadgeProps extends PillsProps {
+  isMultiSig?: boolean;
```

Comment:
This prop name is a bit too tied to its use case. As this is a global component can we try to find another solution? Maybe we could just pass the icon?